knowledge/highways/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: geography
+seed_examples:
+- answer: | The maximum speed limit is 75 mph. But most of the states have a maximum speed limit of 70 mph.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!